app/config/AppConfig.scala

```diff
@@ -22,11 +22,16 @@ import uk.gov.hmrc.play.config.ServicesConfig
 trait AppConfig {
   def platformFrontendHost: String
   def emailServicePath: String
+  def whitelistedDomains: Set[String]
 }
 
 object AppConfig extends AppConfig with ServicesConfig {
   override val platformFrontendHost = getConfigValueFor("platform.frontend.host")
```

Comment:
Move the behaviour to trait. This will be inline with  our other microservices

Reply:
👍